ci-operator/jobs/openshift/release/openshift-release-master-postsubmits.yaml

```diff
@@ -3,81 +3,47 @@ postsubmits:
   - agent: kubernetes
     branches:
     - ^master$
-    cluster: api.ci
+    cluster: app.ci
     decorate: true
     labels:
-      ci.openshift.io/role: infra-canary
-    name: branch-ci-openshift-release-master-app-ci-apply-canary
+      ci.openshift.io/role: infra
+    name: branch-ci-openshift-release-master-app-ci-apply
     spec:
       containers:
       - args:
         - --config-dir=clusters/app.ci
         - --confirm=true
-        - --kubeconfig=/etc/kubeconfig/kubeconfig
         - --as=
         command:
         - applyconfig
-        env:
-        - name: build01_build01_reg_auth_value
-          valueFrom:
-            secretKeyRef:
-              key: build01_build01_reg_auth_value.txt
-              name: build-farm-credentials
-        - name: build01_ci_reg_auth_value
-          valueFrom:
-            secretKeyRef:
-              key: build01_ci_reg_auth_value.txt
-              name: build-farm-credentials
-        - name: slack_api_url
```

Comment:
For myself, please ignore.
I wonder how you injected the secret for alertManager. But that is my learning to do.

Reply:
If I understand this correctly, this var isn't used: https://github.com/openshift/release/blob/98420605a77bee3b316866b3cc2c185a9e23b3bd/cluster/ci/monitoring/Makefile#L65

Reply:
Got it after reading the make target.